docs/topics/gradle/gradle-compiler-options.md

```diff
@@ -155,7 +155,7 @@ compileKotlin {
 compileKotlin {
     kotlinOptions {
         freeCompilerArgs += "-Xexport-kdoc"
-        kotlinOptions.freeCompilerArgs += "-opt-in=org.mylibrary.OptInAnnotation"
+        freeCompilerArgs += "-opt-in=org.mylibrary.OptInAnnotation"
```

Comment:
Nitpicking: this and above could be merged into the last list

Reply:
The idea was to show how to add one argument and how to add a list of arguments:)